signals/attachments/attachment_bounce_back_read_receipt.yml

```
@@ -0,0 +1,4 @@
+name: "Attachment: Bounce backs & read receipts"
+type: "query"
+source: |
+  any(attachments, .content_type == "message/delivery-status")
```

```suggestion
  any(attachments, .content_type in ("message/global-delivery-status", "message/delivery-status"))
```

We've seen message/deliver-status and global-delivery-status.